main.go

```diff
@@ -60,12 +61,14 @@ func main() {
 	var probeAddr string
 	var configFile string
 	var resourceDirectory string
+	var defaultProfiles string
 	var logAPIRequests bool
 	var printVersion bool
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.StringVar(&configFile, "config-file", "/etc/istio-operator/config.properties", "Location of the config file, propagated by k8s downward APIs")
 	flag.StringVar(&resourceDirectory, "resource-directory", "/var/lib/istio-operator/resources", "Where to find resources (e.g. charts)")
+	flag.StringVar(&defaultProfiles, "default-profiles", "default", "Specifies one or more comma-separated profile names to be applied to each Istio resource")
```

Comment:
We should emphasize these profiles will *ALWAYS* be used and merged with whatever the user puts in spec.profile

Reply:
I've inserted the word "always", but the description now sounds strange to me because of the "always" and "each". I think the previous version was pretty clear, since it stated that the profiles "applied to *each* Istio resource".